docs/users/wallets/keplr.md

```diff
@@ -20,18 +20,8 @@ Add the Keplr browser extension following the instructions on the [Keplr website
 
 Open the Keplr extension on your browser. If you are setting up Keplr for the first time, you can either create a new account or import an existing account. Refer to the [Keplr documentation](https://keplr.crunch.help/getting-started) for further information.
 
-## Connect to Evmos Testnet and Request Funds from the Faucet
+## Connect Keplr to Emvos Mainnet
 
-Once you are signed in to the Keplr extension, visit the [Faucet](https://faucet.evmos.dev/) and click the `Request Funds with Keplr` button. Then approve the both following pop ups `Chain Add Request` and `Request Connection` to add the {{ $themeConfig.project.name }} testnet chain (`evmos_{{ $themeConfig.project.testnet_chain_id }}-{{ $themeConfig.project.testnet_version_number }}`) to Keplr and approve the connection.
+Once you are signed in to the Keplr extension, you can connect the wallet with the Evmos network.
 
-![chain add request](./../../img/keplr_approve_chain.png)
-
-After approval, you can see a transaction confirmation informing you that {{ $themeConfig.project.testnet_denom }} have been successfully transferred to your [evmos address](../../users/technical_concepts/accounts#address-formats-for-clients) on the testnet (only `eth_secp256k1` addresses are supported).
-
-![chain add request](./../../img/keplr_transaction.png)
-
-::: tip
-You can request {{ $themeConfig.project.testnet_denom }} from the [Faucet](./../../developers/faucet) for each address only once every 24h. If you try to request multiple times within the 24h cooldown phase, no transaction will be initiated. Please try again in 24 hours.
-:::
-
-View your account balance either by clicking on the Keplr extension or by [interacting with the node](../../quickstart/interact_node). Note that Faucet might not display the amount of {{ $themeConfig.project.testnet_denom }} transferred by the Faucet, as it might be smaller than the number of decimals displayed in the Keplr extension.
+To connect Keplr to mainnet, visit the Evmos [Dashboard](https://app.evmos.org/). On the dashboard, click `CONNECT WALLET` and select `Keplr`. A popup will prompt you `Requesting Connection` to add the Evmos mainnet chain (`evmos_{{ $themeConfig.project.chain_id }}-{{ $themeConfig.project.version_number }}`) to Keplr and approve the connection.
```

Comment:
Evmos mainnet is already built into Keplr; it is the `Evmos (beta)` network. There's no need for this line, it may confuse users.